src/pages/LdnApplication.tsx

```diff
@@ -539,7 +547,7 @@ class LdnApplication extends Component<{}> {
                       <div className='otherInfoButtonContainer'>
                         <Button variant="contained" color="primary" onClick={() => this.prevStep()}>Back</Button>
                         <span style={{ marginRight: '10px' }}></span>
-                        <Button variant="contained" color="primary" onClick={() => this.updateIssueAndContinue()}>Submit Issue</Button>
+                        <Button variant="contained" color="primary" onClick={() => this.updateIssueAndContinue()}>{otherInfo.some(item => item.value === "") ? "Save Draft" : "Submit Issue"}</Button>
                       </div> :
                       <div className='otherInfoButtonContainer'>
                         <span style={{ marginRight: '10px' }}>The issue is successfully submitted!</span>
```

Comment:
the function goToIssue() is not existing anymore.
can you change this button to an <a> tag redirecting to the issue?